File: Java&Kotlin lecture/src/main/java/com/java/guess/GameFor.java
package com.java.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10) + 1;

        Scanner scanner = new Scanner(System.in);

        int number = 0;

        for(int i = 1; i <= 5; i++){
            System.out.println("Please enter a number("+ i + "/" + 5 + "):");
            number = scanner.nextInt();

            if(number > secret) {
                System.out.println("Lower");
            } else if(number < secret) {
                System.out.println("Higher");
            } else {
                System.out.println("Bingo, number is " + number);
                break;
            }
        }

        System.out.println("The end");
    }
}

File: Java&Kotlin lecture/src/main/kotlin/com/kotlin/guess/GameFor.kt
package com.kotlin.guess

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10) + 1

    val scanner = Scanner(System.`in`)

    var number = 0

    for (i in 1..5) {
        println("Please enter a number($i/5):")
        number = scanner.nextInt()

        if (number > secret) {
            println("Lower")
        } else if (number < secret) {
            println("Higher")
        } else {
            println("Bingo, number is $number")
            break
        }
    }

    println("The end")
}